blog/Gemfile

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+source 'https://rubygems.org'
+ruby '2.3.0'
+
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem 'jekyll", "3.4.1'
+
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem 'minima", "~> 2.0'
+
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
```

Comment:
@rickr Since the blog is still currently using the Minima theme for its basic layout/CSS, I say we leave this comment in for now.  I feel that it helps other devs know where to switch out the theme Gem declaration, for when we eventually replace Minima w/ the Jasper theme (or a custom theme we build to match the styles from the OC re-design). 

I'll fix the inconsistent quotes, though `'` vs `"`